### DISCUSSION TITLE
Error finding provisioning plugin for claim err=storageclass.storage.k8s.io \local-storage\ not found

### DISCUSSION THREAD

**Discussion by redshiva** [rabbitmq/cluster-operator#1727] [Q&A]
Has anyone seen this error before? I used the cluster operator to create a cluster and have been using it without problem. But when I upgraded my Ubuntu server from 22.04 to 24.04, I am now seeing these errors.  

```
kubectl describe storageclass
Name:            local-path
IsDefaultClass:  Yes
Annotations:     kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"storage.k8s.io/v1","kind":"StorageClass","metadata":{"annotations":{},"name":"local-path"},"provisioner":"rancher.io/local-path","reclaimPolicy":"Delete","volumeBindingMode":"WaitForFirstConsumer"}
,storageclass.kubernetes.io/is-default-class=true
Provisioner:           rancher.io/local-path
Parameters:            <none>
AllowVolumeExpansion:  True
MountOptions:          <none>
ReclaimPolicy:         Delete
VolumeBindingMode:     WaitForFirstConsumer
Events:                <none>
```

Did something change from local-storage to local-path?  

```
kubectl version
Client Version: v1.28.13
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3

RabbitMQ V 3.13.2
```

**Comment by lukebakken** [comment]
> upgraded my Ubuntu server from 22.04 to 24.04

If this is the only thing that changed, why do you think this is an issue with the RabbitMQ cluster operator?

**Comment by redshiva** [comment]
Yes the system upgrade is the only thing that changed.  I do not know if it an issue with the cluster operator.  I bring it up because that is how I created the cluster.

**Comment by michaelklishin** [comment]
All changes to this Operator [are documented in release notes](https://github.com/rabbitmq/cluster-operator/releases).

This Operator does not provide or make a lot of specific PV plugin assumptions, as [can be seen in the docs](https://www.rabbitmq.com/kubernetes/operator/using-operator#persistence).

The only recent [statefulset resource change](https://github.com/rabbitmq/cluster-operator/commit/82415be7ae2e972d8badbf099679b4764a34762c) does not seem to be relevant.

A quick search for the error in the title reveals a very large number of discussions not related to this Operator or RabbitMQ.

**Comment by redshiva** [comment]
Thank you for the response!  

Best,

Dave
